src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Boolean.cs

```diff
@@ -12,16 +12,14 @@ public static partial class Utf8Parser
         /// </summary>
         /// <param name="source">The Utf8 string to parse</param>
         /// <param name="value">Receives the parsed value</param>
-        /// <param name="bytesConsumed">On a successful parse, receives the length in bytes of the substring that was parsed </param>
-        /// <param name="standardFormat">Expected format of the Utf8 string</param>
+        /// <param name="bytesConsumed">On a successful parse, receives the length in bytes of the substring that was parsed</param>
+        /// <param name="standardFormat">Expected format of the Utf8 string. Supported formats are G and l.</param>
         /// <returns>
         /// true for success. "bytesConsumed" contains the length in bytes of the substring that was parsed.
         /// false if the string was not syntactically valid or an overflow or underflow occurred. "bytesConsumed" is set to 0.
         /// </returns>
         /// <remarks>
-        /// Formats supported:
-        ///     G (default)   True/False
-        ///     l             true/false
+        /// The parsing is case insensitive and the format parameter is not currently used.
```

Comment:
I'm not sure this is entirely accurate. The format is used in that we validate it is a supported format. However, they are treated identically and thus is always case insensitive

Reply:
I updated the doc. Please let me know what you think.